test/unit/services/azurerediscache/provision-spec.js

```diff
@@ -60,6 +60,27 @@ describe('RedisCache - Provision - PreConditions', function() {
     });
 });
 
+describe('RedisCache - Provision - PreConditions incorrect', function() {
+    var validParams = {};
+    var cp;
+        
+    before(function() {
+        validParams = {  /* missing parameters!! */
+            instance_id : 'b259c5e0-7442-46bc-970c-9912613077dd',            
+            azure : azure,
+            provisioning_result: '{\"provisioningState\":\"Creating\"}'
```

Comment:
Got it. I thought the case was for incomplete parameters. For example, there is only `location` in `parameters` and others (e.g. `sku`) are missing.
If the case is for missing parameters, that's fine. Thanks for explaining it.